### ISSUE TITLE
tf.estimate quickstart

### ISSUE THREAD

**Issue by reedkotler** [tensorflow/tensorflow#11783]
I am in the process of making jupyter notebooks from the tensorflow documentation.
( https://github.com/reedkotler/tensorflow-notebooks )

The tensorflow.org page on tf.esimator quickstart (https://www.tensorflow.org/get_started/tflearn)  had numerous problems. 

I've straightened out the coding issues and the fixes are in my notebook https://github.com/reedkotler/tensorflow-notebooks/blob/master/get_started/estimator/estimator.ipynb

I still have some more markup text to straighten out.

Aide from the python2/3, tensoflow 1.2, etc. issues, the actual test case was not predicting properly. I made my own simple one and it predicted just fine so I did not spend time trying to figure out whether the original was wrong or not.

I'm not volunteering to fix the main tensorflow markdown because I'm against the idea of having so much code there that can't be tested and is constantly breaking because tensorflow is changing and so I don't want to try and fix that with no way to test it and to give people something bulletproof that will work. There are lots of posts going unanswered in stackoverflow for 6 months of more on many of these pages, and for this scikit learn one especially because people know that already and it's an obvious starting place for newbies.

**Comment by reedkotler** [comment]
The Iris test cases for training/test worked fine, just the fake "new samples" was not predicting as the web page expected. But my three new samples were all classified correctly.

**Comment by reedwm** [comment]
I'm not quite sure what the problem is. The tutorials aren't designed to be converted to jupyter notebooks, so there will be issues to work out when converting a tutorial to one.

**Comment by reedkotler** [comment]
There are some samples that were not in the training or test set. In the original comments, the two samples are supposed to be iris's of type 1,2 but it predicts to two of type 1. When I make up sample data by looking at the real data, it predicts correctly. But all the self checking from the test and training set show the same predicted accuracy as mentioned in the comments and it's near 100%.

**Comment by reedwm** [comment]
Good catch! Running the tutorial code, I sometimes get the class predictions being [1, 2] (as claimed in the tutorial) but sometimes they are [1, 1].

I think this issue was already fixed (perhaps unintentionally). When I run the [tutorial code in the repo](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/docs_src/get_started/estimator.md), I always get [1, 2] as claimed (I ran 100 times). Once the tensorflow.org is updated for TensorFlow 1.3, the tensorflow.org tutorial should be fixed.
